File: Data Structures & Algorithms/longest-consecutive-sequence/submission-2.java
class Solution {
    public int longestConsecutive(int[] nums) {
        int longest = 0;

        Set<Integer> seen = new HashSet<>();

        for (int n : nums) {
            seen.add(n);
        }

        for (int n : seen) {
            if (!seen.contains(n - 1)) {
                int start = n + 1;

                while (seen.contains(start)) {
                start++;
                }
                longest = Math.max(longest, start - n);
            }
            
        }
        return longest;
    }
}

File: Data Structures & Algorithms/maximum-subarray/submission-2.py
class Solution:
    def maxSubArray(self, nums: List[int]) -> int:
        res = nums[0]
        
        for i in range(len(nums)):
            currSum = 0
            for j in range(i, len(nums)):
                currSum += nums[j]
                res = max(res, currSum)
        return res

